web/src/main/java/org/apache/shiro/web/filter/authc/HttpMethodsExtractor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.shiro.web.filter.authc;
+
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+
+public class HttpMethodsExtractor {
+
+    public static final String PERMISSIVE = "permissive";
+
+    /**
+     * HTTP Authorization header, equal to <code>Authorization</code>
+     */
+    protected static final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
Remove this. Make `HttpAuthenticationFilter`.AUTHORIZATION_HEADER package-private and use that

Reply:
private static final String AUTHORIZATION_HEADER = HttpAuthenticationFilter.AUTHORIZATION_HEADER;
did you mean like this?

Reply:
no, just make the original declaration in `HttpAuthenticationFilter` package-private,
import / use it, don't duplicate it.

Reply:
I accidentally put that in there, even though I wasn't using those variables. So, I deleted those.